llvm/include/llvm/ProfileData/SampleProf.h

```diff
@@ -1091,18 +1091,14 @@ class FunctionSamples {
     return getCanonicalFnName(F.getName(), Attr);
   }
 
-  /// Name suffixes which canonicalization should handle to avoid
```

Comment:
pretty much a drive-by comment: preserve the comments (handle name to avoid profile mismatches) in a way